### PR TITLE
Compare correct commits for Pull Request from a fork

### PR DESCRIPTION
**Pull request by rmhartog**

### CODE REVIEW

**Comment by anandsaw** [comment]
WHAT THE FOX SAY?
buggy-buggy-buggy-ho!
buggy-buggy-buggy-ho!
buggy-buggy-buggy-ho!
WHAT THE FOX SAY?
Joff-tchoff-tchoff-tchoffo-tchoffo-tchoff!
Joff-tchoff-tchoff-tchoffo-tchoffo-tchoff!
Joff-tchoff-tchoff-tchoffo-tchoffo-tchoff!

**Comment by anandsaw** [comment]
I see that the author is not very responsive on this thread...clearly this is a dead repo.

File: src/main/java/nl/tudelft/ewi/sorcerers/resources/RepoResource.java [hunk 1]
```diff
@@ -24,32 +27,30 @@
 	@Context UriInfo uriInfo;
```
Comment (rmhartog) [inline src/main/java/nl/tudelft/ewi/sorcerers/resources/RepoResource.java]: this is a test comment!

Comment (rmhartog) [inline src/main/java/nl/tudelft/ewi/sorcerers/resources/RepoResource.java]: this is a test comment!

Comment (rmhartog) [inline src/main/java/nl/tudelft/ewi/sorcerers/resources/RepoResource.java]: this is a test comment!

File: src/main/java/nl/tudelft/ewi/sorcerers/model/WarningService.java [hunk 2]
```diff
@@ -39,4 +46,15 @@ public Warning addWarningIfNew(Warning warning) {
 	public Warning get(String repo, String commit, Integer warningId) {
 		return this.warningRepository.get(repo, commit, warningId);
 	}
+
+	public Diff getWarningsForDiff(String repo, String base,
+			String head) throws IOException {
+		RepositoryCommitCompare compare = commitService.compare(RepositoryId.createFromId(repo), base, head);
+		String baseCommit = compare.getBaseCommit().getSha();
+
```
Comment (rmhartog) [inline src/main/java/nl/tudelft/ewi/sorcerers/model/WarningService.java]: this is a test comment!

File: src/main/java/nl/tudelft/ewi/sorcerers/github/CommitServiceFactory.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+		this.client = client;
+	}
+
+	@Override
+	public CommitService provide() {
+		return new CommitService(this.client);
+	}
+
+	@Override
+	public void dispose(CommitService instance) {
```
Comment (rmhartog) [inline src/main/java/nl/tudelft/ewi/sorcerers/github/CommitServiceFactory.java]: *Auto-generated comment*
    Parameter instance should be final.

**Pull request merged**